src/words/tweet_tokenizer.jl

```diff
@@ -184,8 +174,8 @@ function html_entity(ts::TokenBuffer, remove_illegal=true)
       return true
     end
```

Comment:
i just noticed that 
all the indentin in this file is wrong.
julia code should be indented with 4 spaces.

Reply:
I noticed 2 space indentation in fast.jl, so I reflected the changes in this as well. 
I will change the indentation in this to 4 spaces and also send a patch fixing indentation in fast.jl

Reply:
Thank you.